src/compute/src/render/context.rs

```diff
@@ -134,8 +134,8 @@ where
 
 impl<S: Scope, T> Context<S, T>
 where
-    T: Timestamp + Lattice,
-    S::Timestamp: Lattice + Refines<T>,
+    T: Timestamp + Lattice + Columnation,
+    S::Timestamp: Lattice + Refines<T> + Columnation,
```

Comment:
I was not completely clear to me whether these trait bounds are absolutely required here? We are forcing `CollectionBundle` to require `Columnation` on its timestamp types, but we could require these bounds only in the implementation including `ensure_collections`?

Another way to phrase the question more conceptually: Do we want to ensure `Columnation` for timestamp types in connection with arrangement usage only or across all contexts (arrangements, dataflow edges) in this PR? The difference is not large now in that the timestamps used in arrangements and the ones in dataflow edges are today the same, but changing the requirements expresses an opinion that they should at least share some more behavior than we've required so far.

Reply:
Unfortunately, we need the bound on `CollectionBundle` because `ArrangementFlavor` needs it. We use both `T` and `S::Timestamp` in the handles, so the `Columnation` bound propagates everywhere.